### DISCUSSION TITLE
Setup token invalid

### DISCUSSION THREAD

**Discussion by unzip0** [envault/envault#44] [Q&A]
**Hi all!**
I'm stucked when i try to init my envault environment. I just created my first app and i copied the command to init the environment : `npx envault env.amos-am.box 51 CP8gRw6WQXfCI4PQ`
I get the next error: 
`Welcome to Envault! No more .env update nightmares from now on, we promise 🤗
Connecting to your Envault server... !
 ›   Error: Looks like your setup token is invalid, please get another!`

When i have to set up the token? maybe i missed something in the configuration..

Thanks !

**Comment by danharrin** [comment]
The setup command displayed in the dashboard expires periodically to prevent brute force attacks. Please reload the app page and ensure that you're using a fresh setup command.

**Reply by unzip0** [reply]
I'm getting same error after reload...

**Reply by unzip0** [reply]
Any idea? any way to debug envault command?

**Reply by jratzenboeck** [reply]
I have the same problem although I followed the instructions step by step. Currently I also don't have an idea what might be the problem.

**Reply by jratzenboeck** [reply]
ok, I could resolve the problem which had nothing to do with envault but rather was an ops issue that there was basic auth enabled in Nginx for the site which caused the api call to envault return a 401.

**Reply by unzip0** [reply]
Basic auth is not enabled in the site... @jratzenboeck `npx envault` is working now for you?

**Reply by jratzenboeck** [reply]
I managed to get it working now although it is tricky with a self-signed SSL certificate like I have it because the request in a Node.js context did not work. However, when I tried the same request in postman it delivered a 200. My solution was to set the NODE_TLS_REJECT_UNAUTHORIZED to 0. There is a warning then when running the command but it works.
